docs/contributing-to-airbyte/issues-and-requests.md

```diff
@@ -1,23 +1,13 @@
-# Issues and Requests
+# Issues and Feature Requests
 
-## Report a Bug
+#### Report a Bug
 
-Bug reports help us make Airbyte better for everyone. We provide a preconfigured template for bugs to make it very clear what information we need.
+Bug reports help us make Airbyte better for everyone. We provide a preconfigured template for bugs to make it very clear what information we need. Check if the bug was [already reported](https://github.com/airbytehq/airbyte/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fbug) before raising a new one to make sure you're not raising a duplicate.
```

Comment:
```suggestion
Bug reports help us make Airbyte better for everyone. We provide a preconfigured template for reporting bugs to make it very clear what information we need. To avoid duplicate reports, check if the bug was [already reported](https://github.com/airbytehq/airbyte/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fbug) before raising a new one. 
```